packages/utilities/src/dom/getRootDocument.ts

```diff
@@ -0,0 +1,15 @@
+import { getWindow } from './getWindow';
+
+/**
+ * Helper to get the top-most document object.
+ *
+ * @public
+ */
+export function getRootDocument(): Document | undefined {
+  let current = getWindow();
+  while (current!.parent !== current) {
```

Comment:
It's generally a good idea to avoid the use of the non-null assertion operator. Convention is to just wrap this in an `if` check.

Reply:
Yes, makes sense, since it could be undefined when SSR is active. See 81118f18c2a60b3e4bd9ea8ce2c762e66a0d9fc1.